src/backend/api/Fusion.Resources.Api/Controllers/Requests/ApiModels/ApiRequestComment.cs

```diff
@@ -1,21 +1,34 @@
-﻿using System;
+﻿using Fusion.Resources.Domain;
+using System;
 using System.Text.Json.Serialization;
 
 namespace Fusion.Resources.Api.Controllers
 {
     public class ApiRequestComment
     {
+        public ApiRequestComment(QueryRequestComment queryComment)
+        {
+            Created = queryComment.Created;
+            CreatedBy = new ApiPerson(queryComment.CreatedBy);
+            Updated = queryComment.Updated;
+
+            UpdatedBy = queryComment.UpdatedBy != null ? new ApiPerson(queryComment.UpdatedBy) : null;
+            Content = queryComment.Content;
+
+            if (Enum.TryParse<ApiCommentOrigin>(queryComment.Origin, true, out var result))
```

Comment:
Should also have a "Unknown" value in the enum to support when the parsing fails. Otherwise the "Company" will be used as that is index 0 and enum is not nullable.